src/main.rs

```diff
@@ -121,6 +146,10 @@ fn get_user_input(msg: &str, secret_number: u32, locale: &Localization) -> u32 {
 			}
 		}
 
+		if commands::check_for_valid_command_name(&user_input) {
```

Comment:
Just saw this, I think this could be called check_command_name() or validate_command_name()

Reply:
Well, actually, I accidentally commited that. I would commit it on another branch. Imma fix it and when commit it, name it `validate_command_name`